src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -64,7 +64,7 @@
     "@types/file-saver": "2.0.1",
     "async-mutex": "0.2.4",
     "bootstrap": "5.2.3",
-    "carbon-components-angular": "5.25.1",
+    "carbon-components-angular": "5.26.0",
```

Comment:
lets do that in a separate PR or commit

Reply:
if we remove this code from json file current changes will break so it will go with same Pr

Reply:
Bumping the version can be in a separate PR/commit as its a significant change.
And we must check that it does not bring any breaking chnages,
Additionally, conflicting with this PR https://github.com/ceph/ceph/pull/60560/files#diff-07cc786d1834b5fa54f059bfee594cb2c5af61b2de7b17afdcbe7166b179c979R66

So if we need a higher version for the component used here, lets stick to one and bump.

cc @nizamial09 @ivoalmeida

Reply:
@pujaoshahu I can upgrade carbon version on my PR to match yours

Reply:
Thank you Ivo